code/datums/gamemode/role/vampire_role.dm

```diff
@@ -2,6 +2,8 @@
  -- Vampires --
  */
 
+#define MAX_BLOOD_PER_TARGET 400
```

Comment:
So a vamp will go straight to jaunt and thrall from bloodbagging one person?

Reply:
Maybe it's a bit much, you're right. Perhaps 200 would be better (and closer to the original)